index.html

```diff
@@ -1259,6 +1267,54 @@ <h1 class="text-white">FOSSASIA is the annual OpenTechSummit featuring prominent
         <script src="js/tweetsfetcher.js"></script>
         <script src="js/placeholders.min.js"></script>
         <script src="js/scripts.js"></script>
+        <script>
+        $.ajax({
+	        type : 'GET',
+	        dataType : 'json',
+	        url: 'schedule/speakers.json',
+	        success : function(data) {
+	            // console.log(data);
+	            var names = data.speakers;
+	            var lastSpeakerId = names[names.length - 1].id;
+	            //console.log(lastSpeakerId);
+	            var testIndex = 0;
+	            var speakerString = "";
+	            var blockNameList = ['End of Sessions', 'Lunch', 'Group Photo', 'Social Event', 'Hong Phuc Dang, Justin Lee', '', 'Lightning Talks', 'Science Hacks and Science Hackers']
+	            for (var nameValue = 0; nameValue < names.length; nameValue++) {
+	            	testIndex++;
+	            	if(names[nameValue].id == testIndex && testIndex <= lastSpeakerId){
+	            		var x = names[nameValue].name;
+	            		x = x.replace(/ /g,'')+'.jpg';
+	            		var found = blockNameList.indexOf(names[nameValue].name);
+	            		console.log(found + ' ' + names[nameValue].name);
+	            		if(found === -1) {
+		            		speakerString += "<div class=\"col-md-3 col-sm-6 speaker-column\">"+
+										 		"<div class=\"speaker\">"+
+									    			"<div class=\"image-holder\">"+
+									    				"<img class=\"background-image\" height=\"300\" width=\"300\" alt=\""+x+"\" src=\"img\/"+x+"\" onerror=\"imgError(this)\">"+
+									     				"<div class=\"hover-state text-center preserve3d\">"+
+															"<div class=\"social-links vertical-align\">"+
+																"<a href=\""+names[nameValue].twitter+"\"><i class=\"icon social_twitter\"></i></a>"+
```

Comment:
Don't show twitter icon for those who haven't given a twitter profile. Same goes for LinkedIn.